corelib-definitions/src/main/java/eu/europeana/corelib/definitions/edm/model/metainfo/AudioMetaInfo.java

```diff
@@ -0,0 +1,12 @@
+package eu.europeana.corelib.definitions.edm.model.metainfo;
+
+public interface AudioMetaInfo {
+    public Integer getSampleRate();
```

Comment:
Interfaces do not require the access level as they are always public. Remove public from all methods in this interface.